lib/commands/web.js

```diff
@@ -14,6 +15,50 @@ extensions.getElementHeightMemoized = _.memoize(async function (key, el) {
   return (await this.getNativeRect(el)).height;
 });
 
+extensions.getExtraTranslateWebCoordsOffset = async function () {
+  let offset = 0;
+
+  let iphone = true;
+  try {
+    const useragent = await this.execute('return navigator.userAgent');
+    iphone = useragent.toLowerCase().includes('iphone');
+  } catch (err) {
+    log.warn(`Unable to find device type from useragent. Assuming iPhone`);
+    log.debug(`Error: ${err.message}`);
+  }
+
+  // keep track of implicit wait, and set locally to 0
+  const implicitWaitMs = this.implicitWaitMs;
+
+  // try to see if there has been scrolling
+  try {
+    this.setImplicitWait(0);
+
+    await this.findNativeElementOrElements('accessibility id', 'ReloadButton', false);
+    // reload button found, which means scrolling has not happened
+  } catch (err) {
+    // no reload button, which indicates scrolling has happened
+    try {
+      const el = await this.findNativeElementOrElements('accessibility id', 'URL', false);
+      offset -= await this.getElementHeightMemoized('URLBar', el);
+    } catch (ign) {
+      // no URL elements found, so continue
+    }
+
+    // when scrolling has happened, there is a tick more offset needed
+    offset -= 10;
```

Comment:
rather than putting `10` in a few places, can we give it a meaningful constant name?